cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,26 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    fn process_version(
```

Comment:
```suggestion
    async fn process_version(
```

Need to be async so that it could `await` tokio call

Reply:
Does it need to be async?  There should be nothing outside of some local CPU/memory access in this code-path to cause any delays.

Reply:
ah, the file read is async, so yes, it does.

Reply:
Updated to use `async` and [`contains`][1] in 2baa1f2f -> 970b3183.

[1]: https://github.com/openshift/cincinnati/pull/314#pullrequestreview-503040937